### ISSUE TITLE
Pytorch Model accuracy and precision is decreasing after tflite conversion

### ISSUE THREAD

**Issue by beingnayana** [PINTO0309/PINTO_model_zoo#254]
### Issue Type

Performance

### OS

Ubuntu

### OS architecture

x86_64

### Programming Language

Python

### Framework

PyTorch, ONNX, TensorFlow, TensorFlowLite

### Model name and Weights/Checkpoints URL

[models.zip](https://github.com/PINTO0309/PINTO_model_zoo/files/8589071/models.zip)

### Description

When the pytorch model is converted into tflite its accuracy and perfomance is drastically decreased. the results of pytorch and tflite inference is given.
[pytorch inference.zip](https://github.com/PINTO0309/PINTO_model_zoo/files/8589095/pytorch.inference.zip)
[tflite inference.zip](https://github.com/PINTO0309/PINTO_model_zoo/files/8589096/tflite.inference.zip)
